DQM/L1TMonitor/src/L1TStage2CaloLayer1.cc

```diff
@@ -43,10 +44,43 @@ void L1TStage2CaloLayer1::dqmBeginRun(const edm::Run&, const edm::EventSetup&)
 
 void L1TStage2CaloLayer1::analyze(const edm::Event & event, const edm::EventSetup & es)
 {
+  // Monitorables stored in Layer 1 raw data but
+  // not accessible from existing persistent data formats
+  edm::Handle<FEDRawDataCollection> fedRawDataCollection;
```

Comment:
hi @nsmith- - apologies this PR fell into a black hole for no reason..

there is an unpacker (now....) that does this same thing (if I understand)- please use its output directly here.

Reply:
Huh? The unpacker for CaloLayer1 has existed since it was installed in 2015.
(a packer was not made until recently, because it is of no use outside DQM)
And no, the unpacker doesn't unpack the technical info in the headers of the FED frames, as they have no DataFormat to hold their information, and we wanted to monitor a few things in a simple manner.